<launch>
    <node pkg="traj_server" type="traj_server_node" name="traj_server" output="screen" launch-prefix="">
        
        <param name="max_pos_err_"      value="0.009"/>
        <param name="max_vel_err_"       value="0.1"/>
        <param name="max_ang_err_"    value="0.1"/>
        <param name="max_omega_err_"    value="0.01"/>
        <param name="max_throttle_err_"    value="6"/>
	</node>
</launch>
